### DISCUSSION TITLE
Meeting Minutes 04-04-2024

### DISCUSSION THREAD

**Discussion by pellet** [NeuroTechX/EEG-ExPy#264] [Developer's Meetings]
# Meeting Minutes
## Meeting Information
**Meeting Date/Time:** April 7, 2024
**Attendees:**  @tmorshed @JohnGriffiths @oreHGA @pellet 
**Note Taker:**  @pellet 

## Agenda Items

Item | Description
---- | ----
Catchup | 6 mins.
Braindecode integration | 17 mins.
EEG-ExPy Installation options | 7 mins.
VR updates | 30 mins.

## Discussion Items
Item | Who | Notes | Attritbuted time |
---- | ---- | ---- | --- |
Braindecode | @tmorshed   | - Looking at creating some notebooks which use the [Braindecode](https://github.com/braindecode) library.<br> - First step could be to analyse N170 ERP data with Braindecode integration. | 17 mins.
EEG-ExPy Installation options | @JohnGriffiths | -Three installation types to be available: Streaming, stimulus presentation and docs build. | 7 mins.
VR functionality | @pellet | -Documentation added into 'Running experiment' section.<br> -Latency concerns regarding alteration of rendering code needed to add support for VR.<br> -VR caveats and limitations. | 30 mins.

## Action Items
Item | Responsible | Due Date |
---- | ---- | ---- |
EEG-ExPy installation options to be finished off | @JohnGriffiths  | N/A
finish setting up windows install for VR testing | @JohnGriffiths | N/A
VR caveats docs to be added | @pellet  | N/A

**Comment by tmorshed** [comment]
P.S. John was able to get the Quest headset to work with the regular 3.0 cable btw - seems like there was an "auto power saving" setting for the driver by default lol

Kudos to him for patiently finding the fix!

**Reply by pellet** [reply]
awesome, nice work :) sounds like a frustrating issue!
